File: redsheet-portal/src/app/database-models/v-avoid-gesture.ts
import { required, maxLength, range } from '@rx/annotations';

export class vAvoidGesture {
    constructor(vAvoidGesture?: vAvoidGesture )  {
        let properties = [ "avoidGestureId", "avoidGestureValue", "cultureCountryId",];
        for (let property of properties)
            if (vAvoidGesture && vAvoidGesture[property])
                this[property] = vAvoidGesture[property];
    }
 
    @range(1,2147483647)
	avoidGestureId : number =   undefined;
 
    @required()
    @maxLength(500)
	avoidGestureValue : string =   undefined;
 
	cultureCountryId : number =   0 ;
}

File: RedSheet.DbEntities/Models/vAvoidGesture.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vAvoidGestures", Schema = "dbo")]
	[Serializable]
	public partial class vAvoidGesture  
	{
 
	[Range(1, int.MaxValue)]
	public int AvoidGestureId { get; set; } 
 
	[Required] 
	[MaxLength(500)]
	public string AvoidGestureValue { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int CultureCountryId { get; set; } 

	public vAvoidGesture()
	{
    }

    }
}

